content/05.discussion_and_conclusion.md

```diff
@@ -1,6 +1,15 @@
 # Discussion and Conclusion
 
-1. We provided a website resource to allow users to see toekn association changes
-2. Word2vec is unstable and we implemented an approach to account for that variation
-3. Constructs groundwork for future research into token changes
-4. Will implement biorxiv and other preprint resources as a next step.
+Language is rapidly evolving, and the usage of words changes over time [@doi:10.1093/acrefore/9780199384655.013.323].
+Words assimilate new meanings or associations.
+Some efforts have been made to study semantic change using biomedical text [@doi:10.1142/9789811232701_0011;@arxiv:2102.07836; @doi:10.2196/22635]; however, no such work has examined the changes evident in both pre-publication peer reviewed and preprinted biomedical text.
```

Comment:
```suggestion
Some efforts have been made to study semantic change using biomedical text [@doi:10.1142/9789811232701_0011;@arxiv:2102.07836; @doi:10.2196/22635]; however, no such work has examined the changes evident in both pre-publication peer-reviewed and preprinted biomedical text.
```
Realized this needs a hyphen - after this merges can you search for "peer reviewed" and make sure that they've got a hyphen if necessary?